### PR TITLE
Added answer question in registration

### PR DESCRIPTION
**Pull request by duckmoll**
#64 
Added answer question in profile creation.
Co-created question and answer_question models with other models.
Added registration test for question and answer_question models.
Below is response from registration:
screen shot 2018-12-16 at 11 54 12 am

### CODE REVIEW

**Comment by duckmoll** [comment]
> Is it the plan that users can create their own question? Or if we want users to choose from pre-defined questions to answer?

Hinge allows users to choose questions to answer or self-define question to answer.

File: new_dawn_server/users/api/resources.py [hunk 1]
```diff
@@ -284,6 +284,33 @@ def get_and_save_city_pref(bundle):
                 location_list.append(city_pref)
         return location_list
 
+    @staticmethod
+    def get_and_save_question(bundle):
+
+        if not Question.objects.filter(question=bundle['question']).exists():
```
Comment (tangziyi001) [inline new_dawn_server/users/api/resources.py]: Let's add a bool field in data model to reflect if the question is user-defined. And make that true when question is created here.

File: new_dawn_server/users/api/tests/test_user_answer_question_registration.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        self.profile_arguments = {
+            "description": "nice",
+            "employer": "MANMAN",
+            "hometown": "NY",
+            "job_title": "CEO",
+            "profile_photo_url": "www",
+            "school": "NYU",
+            "smoke": True,
+        }
+        self.answer_question_arguments = {"answer_question":
```
Comment (tangziyi001) [inline new_dawn_server/users/api/tests/test_user_answer_question_registration.py]: Let's also create a Question object (assuming the question object is created by admin) and test if you can get the same object when storing answer question.

**Comment by tangziyi001** [comment]
Actually ignore my last comment.
You don't need to add a prepend API for that. It can just get from tastypie GET api with filter. So you just need to enable filter on user defined field.

**Pull request merged**